charts/refarch-template/charts/backend/values.yaml

```diff
@@ -0,0 +1,99 @@
+replicaCount: 1
+
+image:
+  registry: ghcr.io
+  repository: it-at-m/refarch/refarch-backend
```

Comment:
hier liegt ein Beispiel Image: https://github.com/it-at-m/sps

`ghcr.io/it-at-m/sps/sps-backend:latest`

Reply:
verstehe den Kommentar nicht. Das verweist auf ein Repo, nicht auf das Image. Im ReadMe des Repos wird auch nicht auf das Image verwiesen. I